src/api/models/article.js

```diff
@@ -0,0 +1,57 @@
+import slug from 'slug';
+
+const THIRTY_SIX = 36;
```

Comment:
@MemunaHaruna gave feedback on this @murediane .
Use variable names that reflect what you used them for